cmd/zed/manage/compact.go

```diff
@@ -0,0 +1,86 @@
+package manage
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/brimdata/zed/cli/commitflags"
+	"github.com/brimdata/zed/cli/lakeflags"
+	apicompact "github.com/brimdata/zed/cmd/zed/manage/compact"
+	"github.com/brimdata/zed/lake/api"
+	"github.com/brimdata/zed/pkg/charm"
+	"gopkg.in/yaml.v3"
+)
+
+var compact = &charm.Spec{
+	Name:  "compact",
+	Usage: "compact",
+	Short: "compact objects in a pool",
+	New:   newCompact,
+}
+
+type compactCommand struct {
+	*Command
+	commitFlags commitflags.Flags
+	coldthresh  time.Duration
+	config      Config
+}
+
+func newCompact(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &compactCommand{Command: parent.(*Command)}
+	c.commitFlags.SetFlags(f)
+	f.Func("config", "path of manage yaml config file", func(s string) error {
```

Comment:
Any reason not to make this a required positional argument?

Reply:
I wanted to have the ability to override config file settings with command line flags and this makes it possible.